CPP-Example.cpp

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <string.h>
+
+int main()
+{
+   char src[40];
+   char dest[100];
+   char dest2[100];
+
+   gets(src); 
+
+   strcpy(dest, src); 
+ 
+   strncpy(dest,src, strlen(dest)); //DevSkim: ignore DS111237,DS154189 
+   
+   strcat(dest, "this is bad");
```

Comment:
[Banned C function detected (strcat)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS141863.md)
If the combination of strings is larger than the destination buffer, strcat will cbuffer overflow the destination buffer

Suggestions:
**[0]** *+   strcat_s(dest, <size of dest>,  "this is bad");*
**[1]** *+   strlcat(dest,  "this is bad", <size of dest>);*